test/auro-nav.test.js

```diff
@@ -0,0 +1,44 @@
+import { fixture, html, expect } from '@open-wc/testing';
+import '../index.js';
+
+describe('auro-nav', () => {
+  it('auro-nav is accessible', async () => {
+    const el = await fixture(html`
+      <auro-nav cssclass="testClass"></auro-nav>
+    `);
+
+    await expect(el).to.be.accessible();
+  });
+
+  it('auro-nav custom element is defined', async () => {
```

Comment:
```suggestion
  it('auro-nav custom element is defined using breadcrumb links', async () => {
```

let's make the test unique so that it still makes sense when we add new styles later